lib/hiera/backend/module_data_backend.rb

```diff
@@ -47,17 +47,16 @@ def load_data(path)
         end
       end
 
-      def no_answer
```

Comment:
ok, in an effort to keep the change as small as possible please do not rename this method